### PR TITLE
Update client protocol document

### PR DESCRIPTION
**Pull request by starwarfan**

### CODE REVIEW

File: doc/Client-Portal Protocol.md [hunk 1]
```diff
@@ -195,30 +209,6 @@ This a format for client reconnects.
...
-            object(VideoInfo)::
-              {
-               status: "active" | "inactive" | undefined,
-               source: "camera" | "screen-cast" | "raw-file" | "encoded-file" | undefined,
-               original: [{
-                 format: object(VideoFormat),
-                 parameters: object(VideoParameters) | undefined,
-                 simulcastRid: string(SimulcastRid) | undefined
-               }],
-               optional:
```
Comment (starwarfan) [inline doc/Client-Portal Protocol.md:207]: Or keep "optional" in each tracks

File: doc/Client-Portal Protocol.md [hunk 2]
```diff
@@ -172,20 +188,18 @@ This a format for client reconnects.
...
+              tracks: [object(TrackInfo)]
             }
 
-            object(AudioInfo)::
+            object(TrackInfo)::
               {
-               status: "active" | "inactive" | undefined,
-               source: "mic" | "screen-cast" | "raw-file" | "encoded-file" | undefined,
-               format: object(AudioFormat),
-               optional:
```
Comment (starwarfan) [inline doc/Client-Portal Protocol.md:184]: Add optional and mid,rid

File: doc/Client-Portal Protocol.md [hunk 3]
```diff
@@ -431,40 +419,36 @@ This a format for client reconnects.
...
-        object(VideoParametersSpecification)::
+       tracks: [
           {
-           resolution: object(Resolution) | undefined,
-           framerate: number(WantedFrameRateFPS) | undefined,
-           bitrate: number(WantedBitrateKbps) | string(WantedBitrateMultiple) | undefined,
-           keyFrameInterval: number(WantedKeyFrameIntervalSecond) | undefined
+            type: "audio" | "video",
+            mid: string(MID),
+            from: string(TrackID),
```
Comment (starwarfan) [inline doc/Client-Portal Protocol.md:432]: publication+trackId / mid/rid

Comment (starwarfan) [reply doc/Client-Portal Protocol.md:432]: TrackID

File: doc/Client-Portal Protocol.md [hunk 4]
```diff
@@ -158,6 +158,22 @@ This a format for client reconnects.
          views: [string(ViewLabel)],
          streams: [object(StreamInfo)],
          participants: [object(ParticipantInfo)]
+         transcodingCapability: {
```
Comment (starwarfan) [inline doc/Client-Portal Protocol.md:161]: Put transcodingCapability into track's optional

**Review by lzhai — APPROVED** [review]

**Pull request merged**